terms.html

```diff
@@ -177,17 +187,16 @@
   <dt><dfn>DID URL dereferencing</dfn></dt>
 
   <dd>
-The function that takes as its input a <a>DID URL</a>
-and a set of input metadata, and returns a <a>resource</a>. This
-resource might be a <a>DID document</a> plus additional metadata, or it might be a
-secondary resource contained within the <a>DID document</a>, or it might be a
-resource entirely external to the <a>DID document</a>. The function
-uses the <a>DID resolution</a> function to fetch a
-<a>DID document</a> indicated by the <a>DID</a> contained within the
-<a>DID URL</a>. The dereferencing function then can perform additional
-processing on the <a>DID document</a> to return the dereferenced resource
-indicated by the <a>DID URL</a>. The inputs and outputs of this function are
-defined in <a href="#did-url-dereferencing"></a>.
+The function that takes as its input a <a>DID URL</a> and a set of input
```

Comment:
Do you also want to change "function" to "process" here, to be consistent with your changes in "DID resolution". I'm fine with either term, but it should be the same for both term definitions.